### PR TITLE
Add exercises/practice/tournament

### LINKED ISSUE #27: Tournament exercise
Claimed

### PR DESCRIPTION
**Pull request by IsaacG**
Closes #27

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -71,6 +71,19 @@
...
+        "name": "Tournament",
+        "uuid": "abc1dd61-d677-4e7c-95a6-b9c6f797b2bd",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 5,
+        "topics": [
+          "arrays",
+          "sorting",
+          "text_formatting"
+        ]
```
Comment (glennj) [inline config.json:85]: We're not using this key in v3.

File: exercises/practice/tournament/.meta/example.awk [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        draw($2)
+        break
+    default:
+        exit(1)
+    }
+}
+
+END {
+    print "Team                           | MP |  W |  D |  L |  P"
+    PROCINFO["sorted_in"] = "sort_order"
```
Comment (glennj) [inline exercises/practice/tournament/.meta/example.awk:65]: Yes!

**Review by glennj — APPROVED** [review]
I knew this would be an excellent awk exercise. Nicely done!

File: exercises/practice/tournament/.meta/example.awk (hunk 2 above)
Comment (IsaacG) [reply exercises/practice/tournament/.meta/example.awk:65]: This exercise was harder than I anticipated. My awk knowledge is pretty basic and it took a fair bit of work to figure it all out :D

File: config.json (hunk 1 above)
Comment (IsaacG) [reply config.json:85]: Meh. Already merged. Should I drop this in another PR?

File: exercises/practice/tournament/.meta/example.awk (hunk 2 above)
Comment (glennj) [reply exercises/practice/tournament/.meta/example.awk:65]: Woot! You're a hero ;)

**Comment by ErikSchierboom** [comment]
> I knew this would be an excellent awk exercise. Nicely done!

Wow, the solution is surprisingly concise!

**Pull request merged**